dvc/repo/tree.py

```diff
@@ -428,11 +428,12 @@ def copytree(self, top, dest):
 
         for root, _, files in self.walk(top):
             root = PathInfo(root)
-            makedirs(dest, exist_ok=True)
+            dest_dir = dest / root.relative_to(top)
+            makedirs(dest_dir, exist_ok=True)
             for fname in files:
                 src = root / fname
                 with self.open(src, mode="rb") as fobj:
-                    copy_fobj_to_file(fobj, dest / fname)
```

Comment:
`dest` here was the top level destination path, rather than the expected relative path

Reply:
Yeah, I should not have removed that without thinking once. My bad.